test/testdata/cfg/examples.rb.cfg.exp

```diff
@@ -170,7 +170,7 @@ subgraph "cluster_::Examples#variables" {
 
     "bb::Examples#variables_6" -> "bb::Examples#variables_7" [style="bold"];
     "bb::Examples#variables_7" [
-        label = "block[id=7](a: Integer(1), b: Integer(2))\l<returnMethodTemp>$2: Integer = a: Integer(1).+(b: Integer(2))\l<finalReturn>: T.noreturn = return <returnMethodTemp>$2: Integer\l<unconditional>\l"
+        label = "block[id=7](b: Integer(2), a: Integer(1))\l<returnMethodTemp>$2: Integer = a: Integer(1).+(b: Integer(2))\l<finalReturn>: T.noreturn = return <returnMethodTemp>$2: Integer\l<unconditional>\l"
```

Comment:
Are we missing a fast_sort somewhere in our printing code? Or should we expect that these files will have changed?

See all the CFG exp changes in this commit: https://github.com/sorbet/sorbet/pull/1125/commits/a8e38a32967c3d02c9f6415164f3495e90235494

Reply:
https://github.com/sorbet/sorbet/pull/1125#issuecomment-506604516